inabox/tests/setup_disperser_harness.go

```diff
@@ -35,12 +47,16 @@ type DisperserHarnessConfig struct {
 	BucketTableName     string
 	S3BucketName        string // S3 bucket name for blob storage
 	MetadataTableNameV2 string
+	OnDemandTableName   string // DynamoDB table name for on-demand payments
```

Comment:
mention that this is used by controller

Reply:
https://github.com/Layr-Labs/eigenda/pull/2130/commits/4c74da3be4fb6dd0c465c11fa3cfb6af284a8356